lib/Runtime/Library/CustomExternalIterator.h

```diff
@@ -104,11 +104,10 @@ namespace Js
     public:
         CustomExternalIterator(DynamicType* type, ExternalIteratorKind kind, JavascriptTypeId typeId, NextFunction nextFunction);
 
-        static bool Is(Var aValue);
-        static CustomExternalIterator* FromVar(Var aValue);
-        static CustomExternalIterator* UnsafeFromVar(Var aValue);
         static Var CreateNextFunction(JavascriptLibrary *library, JavascriptTypeId typeId);
         static Var EntryNext(RecyclableObject* function, CallInfo callInfo, ...);
     };
 
+    template <> bool VarIs<CustomExternalIterator>(RecyclableObject* obj);
```

Comment:
Also curious why this template is only declared but not defined here.

Reply:
There are a couple of others too; If there's no particular reason, I'd prefer if the definitions were all in headers, both for simplicity and because I think it better enables inlining.